notebooks/climate/climate_analysis.ipynb

```diff
@@ -0,0 +1,561 @@
+{
```

Comment:
is this notebook part of the fire analysis updates? doesn't seem to be?

Reply:
excellent point- will remove.